### ISSUE TITLE
Upgrade from Chromium 102.0.5005.61 to Chromium 102.0.5005.78

### ISSUE THREAD

**Issue by mkarolin** [brave/brave-browser#23131]
Minor Chromium bump.

https://chromium.googlesource.com/chromium/src/+log/102.0.5005.61..102.0.5005.78?pretty=fuller&n=10000

QA tests:

Check branding items
Check for version bump

Additional checks:

No specific code changes in Brave (only line number changes in patches).

**Comment by kjozwiak** [comment]
Closing as https://github.com/brave/brave-core/pull/13526 was merged into `1.39.x` as builds created via https://github.com/brave/brave-browser/releases/tag/v1.39.115.

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Samsung S10+` running `Android 12` using the following build(s):

```
Brave | 1.39.115 Chromium: 102.0.5005.78 (Official Build) (64-bit)
--- | ---
Revision | df6dbb5a9fd82af3f567198af2eb5fb4876ef99c-refs/branch-heads/5005_59@{#3}
OS | Android 12; Build/SP1A.210812.016
```

**`Example`** | **`Example`**
---------------|-----------------
![Screenshot_20220529-224453_Brave](https://user-images.githubusercontent.com/2602313/170908248-34beb65d-0768-4af5-8467-eaf7c4cea994.jpg) | ![Screenshot_20220529-224446_Brave](https://user-images.githubusercontent.com/2602313/170908249-f489f056-1c5a-44ce-88bc-f4a1750ffe24.jpg)
